src/charm.py

```diff
@@ -183,19 +175,12 @@ def _make_ingress_config(self) -> list:
     def _make_env_config(self, domain="svc.cluster.local") -> dict:
         """Return dict to be used as as runtime environment variables."""
         config = self.model.config
-        relation = None
-        try:
-            relation = self.model.get_relation("ingress-proxy")
-            if relation:
-                backend_site_name = relation.data[relation.app]["service-hostname"]
-                site = backend_site_name
-        except KeyError:
-            pass
+        relation = self.model.get_relation("ingress-proxy")
         if relation:
+            site = relation.data[relation.app]["service-hostname"]
             svc_name = relation.data[relation.app]["service-name"]
             svc_port = relation.data[relation.app]["service-port"]
-            backend_site_name = relation.data[relation.app]["service-hostname"]
-            site = backend_site_name
+            site = relation.data[relation.app]["service-hostname"]
```

Comment:
`backend_site_name`is actually used in this method:
`"NGINX_BACKEND_SITE_NAME": backend_site_name,`